Upstream: yes

From da0aa13642dfc8d4dbaf1f0f5d223714077c1e50 Mon Sep 17 00:00:00 2001
From: Jonathan Riddell <jr@jriddell.org>
Date: Tue, 1 Apr 2014 15:42:47 +0100
Subject: [PATCH] rename solid-hardware to solid-hardware5 to stop it clashing
 with the equivalent binary from kde-runtime in kdelibs4 land

REVIEW:117208
---
 src/tools/solid-hardware/CMakeLists.txt | 11 ++++-------
 1 file changed, 4 insertions(+), 7 deletions(-)

diff --git a/src/tools/solid-hardware/CMakeLists.txt b/src/tools/solid-hardware/CMakeLists.txt
index 7cb604e..dbda0d4 100644
--- a/src/tools/solid-hardware/CMakeLists.txt
+++ b/src/tools/solid-hardware/CMakeLists.txt
@@ -1,7 +1,4 @@
-add_executable(solid-hardware solid-hardware.cpp)
-ecm_mark_nongui_executable(solid-hardware)
-target_link_libraries(solid-hardware KF5::Solid)
-install(TARGETS solid-hardware ${INSTALL_TARGETS_DEFAULT_ARGS})
-
-
-
+add_executable(solid-hardware5 solid-hardware.cpp)
+ecm_mark_nongui_executable(solid-hardware5)
+target_link_libraries(solid-hardware5 KF5::Solid)
+install(TARGETS solid-hardware5 ${INSTALL_TARGETS_DEFAULT_ARGS})
-- 
1.9.1

